application.py

```diff
@@ -1,16 +1,23 @@
-from flask import Flask, render_template,request
+from flask import Flask, render_template,request, redirect, url_for, session
 from flask_sqlalchemy import SQLAlchemy
 from models import *
 
+
 app = Flask(__name__)
 
 app.config["SQLALCHEMY_DATABASE_URI"] = "postgres://shbhdlrudoukbc:56b95ae93382d47bfe6f63214cd8fdd5bb7353e73867d0c6f1f09d5a390fc60a@ec2-54-165-36-134.compute-1.amazonaws.com:5432/d4e8h9gsq3giaq"
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 db.init_app(app)
 
+# setting up the secret key for the application
+app.config['SECRET_KEY'] = 'f9a1520561f1faf67f36a3a620a45e80'
+
 @app.route('/')
-@app.route('/register')
 def home():
+    return render_template('home.html')
+
+@app.route('/register')
```

Comment:
/register should handle both GET and POST requests. When the request is get it should render the registration page, and on POST it should create a user object and update it to the db if the user is not already existing. After successful user creation user can be redirected to login page or profile page. On failure of creation the user can be redirected back to same /register with an error message

Reply:
I will make the changes and I will let you know @ashishk144 .

Reply:
I have made the changes based on your comments. Could you please review it.